cinema/views.py

```diff
@@ -21,7 +23,7 @@
     MovieSessionDetailSerializer,
     MovieListSerializer,
     OrderSerializer,
-    OrderListSerializer,
+    OrderListSerializer, MovieImageSerializer,
 )
```

Comment:
Code Style issue: Group imports using `()` if needed. Although it's not a critical issue, it's a good practice to group together related imports for better readability and organization. Consider grouping the serializer imports for better readability.